packages/product/src/types/services/product.ts

```diff
@@ -0,0 +1,27 @@
+import { ProductStatus } from "@medusajs/types"
+
+export interface UpdateProductDTO {
```

Comment:
comment: can we add a todo to move the other DTO related to internal services? Or do you plan to do it in a follow up pr?

Reply:
I'll be doing it slowly as I come across other services. I'll do a general cleanup in the end.